webvtt/api/VTTCue/position.html

```diff
@@ -24,5 +25,8 @@
 
     cue.position = 1.5;
     assert_equals(cue.position, 1.5);
+
+    cue.position = 'auto';
```

Comment:
'auto' is the initial value; maybe a buggy implementation would disallow setting it to 'auto' after having set to a number, was my thinking...